rest-api/test/unit_test/dao_test/questionnaire_response_dao_test.py

```diff
@@ -656,8 +656,17 @@ def _get_questionnaire_response_with_consents(self, *consent_paths):
         self.LN_QUESTION.linkId,
         self.EMAIL_QUESTION.linkId,
         consent_paths)
+
+    # We need to remove the unused answers in the resource so we don't trigger an unused
+    # link id exception.
+    res_len = len(resource['group']['question']) - 1
+    for idx in range(res_len, -1, -1):
+      if resource['group']['question'][idx]['linkId'].isdigit() is True:
```

Comment:
nit: drop the `is True`?

Reply:
In python database applications I like to be explicit when test boolean values since its possible that a boolean can be (None, True or False) depending on the underling data.  I get carried away with that check sometimes.